tests/helpers/__init__.py

```diff
@@ -8,8 +8,8 @@ class Closeable(object):
     def __init__(self):
         self._closed = False
 
-    def __del__(self):
-        self.close()
+#    def __del__(self):
```

Comment:
Was this for debugging, or intentional?

Reply:
Yeah, I didn't mean to leave that.  I'll fix it.